src/Simplex/Chat/Messages.hs

```diff
@@ -539,6 +541,7 @@ data RcvGroupEvent
   | RGEMemberDeleted {groupMemberId :: GroupMemberId, profile :: Profile} -- CRDeletedMember
   | RGEUserDeleted -- CRDeletedMemberUser
   | RGEGroupDeleted -- CRGroupDeleted
+  | RGEGroupProfileUpdated -- CRGroupProfileUpdated
```

Comment:
maybe have profile (or both old and new) in CIContent?

Reply:
added